docs/en/query_language/table_functions/generate.md

```diff
@@ -0,0 +1,38 @@
+# generate
+
+Generates random data with given schema.
+Allows to populate test tables with data.
+Supports all data types that can be stored in table except LowCardinality, AggregateFunction.
```

Comment:
We use backticks for emphasizing keywords or links. In this case, links are preferable.